grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -937,6 +937,11 @@ class UrlMappings {
             action = [GET: "getCandidates"]
         }
 
+        "/api/facilities/$facilityId/cycle-counts/requests" {
```

Comment:
the [original design](https://pihemr.atlassian.net/wiki/spaces/OB/pages/3332505612/Cycle+Count+Items+API#Create-Cycle-Count-Request) has: `/api/facilities/{facility_id}/cycle-counts/request/batch` (though now that I look at it, maybe `.../requests/batch` is better)

I did it that way to be more explicit about the fact that this is a batch API call. IMO `/cycle-counts/requests` would be used for simple CRUD APIs on a single CycleCountRequest. But maybe that's just personal preference.

Reply:
I'm fine with adding `/batch`.